### ISSUE TITLE
An overlay button over a camera is to be focused. MaterialTap covers up that overlay button.

### ISSUE THREAD

**Issue by akeshwar** [sjwall/MaterialTapTargetPrompt#9]
It is shown something like this:
![selection_003](https://cloud.githubusercontent.com/assets/10617604/18303422/ab12aa0e-74fa-11e6-85f7-2a0e227bf916.png)

(Emulator camera is being used here)

While ideally, it shouldn't cover the overlay icon at the top left corner, because that icon is the target:
![selection_004](https://cloud.githubusercontent.com/assets/10617604/18303453/ce07feec-74fa-11e6-9ce0-a172ecfad862.png)

Here's what I am doing in my app:
I'm showing the live camera preview with a couple of overlay buttons. Those buttons were added in a separate layout file, control.xml. This is how I added my overlay to the camera:
`viewControl = LayoutInflater.from(getBaseContext()).inflate(R.layout.control, null);`

And then targeted one of the icons like this:

    profileIcon = (ImageView) viewControl.findViewById(R.id.profile_icon);
    new MaterialTapTargetPrompt.Builder(CustomCamera.this)
                .setTarget(profileIcon)
                .setPrimaryText("Click Picture")
                .setSecondaryText("Tap here to click the picture of the problem")
                .setOnHidePromptListener(new MaterialTapTargetPrompt.OnHidePromptListener()
                {
                    @Override
                    public void onHidePrompt(MotionEvent event, boolean tappedTarget) {
                    }
                    @Override
                    public void onHidePromptComplete()
                    {
                    }
                })
                .show();

I cloned the project and tried changing a couple of alpha values in the class MaterialTapTargetPrompt, but that served no help.

**Comment by akeshwar** [comment]
using "transparent" as the focal color solved the issue:

    .setFocalColour(Color.TRANSPARENT)

**Comment by sjwall** [comment]
I think the issue maybe that the icon is white against the focals white background.

You could use `builder.setIcon(R.drawable.icon)` or `builder.setIconDrawable(icon)`, this will then draw the supplied drawable on top shaded the same colour as the prompt background. This is how I got around the issue with the hamburger and overflow icons in the action bar in the example app.

**Comment by akeshwar** [comment]
Thanks for the quick support.
I tried `builder.setIcon(R.drawable.icon)`.
I also tried `builder.setFocalColour(Color.TRANSPARENT)`.

But both of them give rise to a new issue in itself:

![selection_005](https://cloud.githubusercontent.com/assets/10617604/18308935/56d0c1c8-7514-11e6-8fc1-bcc6068cf58f.png)

I can see the icon, but, the camera-preview background which is supposed to be around the icon disappears. Besides, using `builder.setIcon(R.drawable.icon)` is giving me an enlarged icon probably because I've some padding there.

The key issue now is the lost-background around the icon.
